recipe/meta.yaml

```diff
@@ -1,27 +1,25 @@
 {% set name = "python3-openid" %}
-{% set version = "3.1.0" %}
+{% set version = "3.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 628d365d687e12da12d02c6691170f4451db28d6d68d050007e4a40065868502
+  sha256: 33fbf6928f401e0b790151ed2b5290b02545e8775f982485205a066f874aaeaf
 
 build:
-  skip: True  # [py2k]
-  number: 1004
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python
+    - python >=3.4
     - pip
   run:
-    - python
-    - defusedxml  # [py>=34]
```

Comment:
My understanding is that this dependency is still needed, it just doesn't need the bit about the python version anymore.

Reply:
oh yes sorry, you are right, fixing it ASAP

Reply:
My understanding of this line is that only the version constraint is a function of python version, not the dependency itself:
https://github.com/necaris/python3-openid/blob/v3.2.0/setup.py#L11

Reply:
Perfect, thanks!